R/formula-tools.R

```diff
@@ -294,18 +297,22 @@ rw <- function(time, by, type = c("independent", "dependent")) {
 #' epinowcast:::construct_rw(rw(week, day_of_week), data)
 construct_rw <- function(rw, data) {
   if (!inherits(rw, "enw_rw_term")) {
-    stop("rw must be a random walk term as constructed by rw")
+    rlang::abort("rw must be a random walk term as constructed by rw")
```

Comment:
```suggestion
    rlang::abort("Argument `rw` must be a random walk term as constructed by `epinowcast:::rw`")
```

or some such. Bit ambiguous currently.